.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "lib/bgfx-cmake"]
 	path = lib/bgfx-cmake
-	url = https://github.com/degenerated1123/bgfx-cmake.git
+	url = https://github.com/Kukunin/bgfx-cmake.git
```

Comment:
You shouldn't change the repository url, please keep it to what it was before

Reply:
I know, I just changed temporarily in this PR, while you didn't merge this into upstream